docs/src/modules/components/DemoContainer.tsx

```diff
@@ -64,8 +93,18 @@ export function DemoContainer(props: PickersGridProps) {
     spacing = { xs: getSpacing('column'), lg: getSpacing('row') };
   }
 
+  if (childrenTypes.has('UI-view')) {
+    sx = null;
+  } else if (childrenTypes.has('single-input-range-field')) {
+    sx = { [`& > .${textFieldClasses.root}`]: { minWidth: 400 } };
+  } else if (childrenSupportedSections.has('date-time')) {
+    sx = { [`& > .${textFieldClasses.root}`]: { minWidth: 250 } };
```

Comment:
Date time range (field) case could still use some style tweaking to avoid the need for scrolling 🤔 🙈 

https://user-images.githubusercontent.com/4941090/206999674-a961f221-8b22-41b8-abe0-f8a53dfa76be.mov

Reply:
I did not use the grid on the `Basic usage` demo
So I could not benefit from those `minWidth`.

I added the grid to fix the scrolling issue
And moved the `minWidth` to `272`

Reply:
Fixed, I'll wait for a last check before merging :+1:

Reply:
I think that `255` would work better and does seem enough based on testing.
Here is an example of `form props` example already using it and it seems enough. 👌

Reply:
I'm trying to stick to multiple of 8, to match with the spacing of MUI
But I can reduce to `256`

Reply:
If I were to further nitpick, [this page](https://deploy-preview-7149--material-ui-x.netlify.app/x/react-date-pickers/date-time-range-field/) still has insufficient width on Chrome. But it seems that Chrome is exception in this case and it might just be a fluke on their side. 🤷

Reply:
I tried to reproduce without success
If you want to do a follow up or create an issue :angel: